plans/smoke.fmf

```diff
@@ -0,0 +1,8 @@
+summary:
+  Basic smoke tests.
+execute:
+  script:
+  - packit --version
+  - packit --help
+  - packit status https://github.com/packit/packit
+  - packit status https://src.fedoraproject.org/rpms/packit
```

Comment:
I'd drop these two since we can run out of quota for un-auth API requests which would result in unrelated test failures